### DISCUSSION TITLE
Add new configuration files for DETR algorithm in mmdetection. 给 mmdet DETR 算法增加新配置文件

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#604] [MMDetection]
# Add new configuration files for DETR algorithm in mmdetection. 
 给 mmdet DETR 算法增加新配置文件 

According to the [Config adaptation document](https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc) and the supported algorithm configurations, add all configurations for DETR and verify the correctness of inference and training.

根据 [Config适配文档](https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc) 内容和已经支持的算法配置，给 DETR 所有配置新增配置并验证推理和训练正确性

## Task Information

**Repository**：MMDetection

**Difficulty**：Easy

**Points**：20

**Estimated Time Consumption**：21 days

## Task Status

**Claimant's Github ID**：zeyuanyin

**Claim Time**：2023年08月04日

**PR Link**：https://github.com/open-mmlab/mmdetection/pull/10785

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by zeyuanyin** [comment]
PR Link：https://github.com/open-mmlab/mmdetection/pull/10785

**Reply by vansin** [reply]
good~
